### ISSUE TITLE
Reposts / include replies

### ISSUE THREAD

**Issue by micressor** [fiatjaf/njump#20]
I have noticed that reposts/replies are not displayed in the profile. could you possibly add a switch there?

**Comment by cxplay** [comment]
As far as I know, the event of replies are included in the user profile. Only retweets are not.

**Comment by micressor** [comment]
> As far as I know, the event of replies are included in the user profile. Only retweets are not.

That's right, it could be.

**Comment by fiatjaf** [comment]
I don't think we should do this. The goal here is not to provide a full client experience, just a simple way for search engines to index Nostr content and for people to get a quick glance of profiles.

**Comment by cxplay** [comment]
> I don't think we should do this. The goal here is not to provide a full client experience, just a simple way for search engines to index Nostr content and for people to get a quick glance of profiles.

So is the Atom feed generated from profiles also affected by this? Honestly, I've enjoyed reading some of the users' timelines in an RSS reader, including their possible replies and reposts.
